backend/btrixcloud/orgs.py

```diff
@@ -402,6 +457,15 @@ async def update_quotas(
 
         return {"updated": True}
 
+    @router.get(
+        "/storage-quota",
+        tags=["organizations"],
+        response_model=OrgStorageQuotaReachedOut,
+    )
+    async def get_storage_quota_reached(org: Organization = Depends(org_dep)):
```

Comment:
The separate endpoint is necessary for polling of the quota and easier access on the frontend, right?
Otherwise, seems like it could be part of the standard org GET api also.